src/server/bridges/ILivechatBridge.ts

```diff
@@ -27,4 +27,6 @@ export interface ILivechatBridge {
     closeRoom(room: ILivechatRoom, comment: string, appId: string): Promise<boolean>;
     findRooms(visitor: IVisitor, departmentId: string | null, appId: string): Promise<Array<ILivechatRoom>>;
     findDepartmentByIdOrName(value: string, appdId: string): Promise<IDepartment | undefined>;
+
+    setCustomFields(data: { token: IVisitor['token']; key: string; value: string; overwrite: boolean }, appId: string): Promise<any>;
```

Comment:
Is it possible for us to determine what should be the return of the method by its implementation on Rocket.Chat? We really should avoid `any` type annotations in the APIs

Reply:
The setCustomFields originally returns **the number of documents affected while performing a `update` operation to the db**. I made some preprocess here: https://github.com/RocketChat/Rocket.Chat.Apps-engine/pull/313/commits/85cf62cd3994dc96127275cc60455706eacc5d82#diff-f859ad2ae3aa3b6f9d373e0f42479597R18 to make its return type to `Promise<boolean>`. Does this preprocess make sense or not? Or

Reply:
IMO having a boolean as the return type makes the method simpler to reason about (and you already did it, right?)

Reply:
yeah, I already changed the signature to `....: Promise<boolean>`.